python/ray/dag/compiled_dag_node.py

```diff
@@ -379,6 +381,22 @@ class CompiledDAG:
     information.
     """
 
+    @ray.remote(num_cpus=0)
+    class DAGDriverProxyActor:
+        """
+        To support the driver as a reader, the output writer needs to be able to invoke
+        remote functions on the driver. This is necessary so that the output writer can
+        create a reader ref on the driver node, and later potentially create a larger
+        reader ref on the driver node if the channel backing store needs to be resized.
+        However, remote functions cannot be invoked on the driver.
+
+        An accelerated DAG creates an instance of this class when the DAG is
```

Comment:
Can you specify that this is a Ray actor (not just an "instance" which could be a local Python instance or a remote instance inside a Ray actor)? Also that the actor needs to be on the same node as the driver.